lambdas/data-migration1/tests/test-collections.js

```diff
@@ -183,16 +183,61 @@ test.serial('migrateCollectionRecord ignores extraneous fields from Dynamo', asy
   await t.notThrowsAsync(migrateCollectionRecord(fakeCollection, t.context.knex));
 });
 
-test.serial('migrateCollectionRecord throws RecordAlreadyMigrated error for already migrated record', async (t) => {
-  const fakeCollection = generateFakeCollection();
+test.serial('migrateCollectionRecord throws RecordAlreadyMigrated error if already migrated record is newer', async (t) => {
+  const fakeCollection = generateFakeCollection({
+    updatedAt: Date.now(),
+  });
 
   await migrateCollectionRecord(fakeCollection, t.context.knex);
+
+  const olderFakeCollection = {
+    ...fakeCollection,
+    updatedAt: Date.now() - 1000, // older than fakeAsyncOp
+  };
+
   await t.throwsAsync(
-    migrateCollectionRecord(fakeCollection, t.context.knex),
+    migrateCollectionRecord(olderFakeCollection, t.context.knex),
     { instanceOf: RecordAlreadyMigrated }
   );
 });
 
+test.serial('migrateCollectionRecord updates an already migrated record if the updated date is newer', async (t) => {
+  const fakeCollection = generateFakeCollection({
+    updatedAt: Date.now() - 1000,
+  });
+  await migrateCollectionRecord(fakeCollection, t.context.knex);
+
+  const newerFakeCollection = generateFakeCollection({
+    ...fakeCollection,
+    updatedAt: Date.now(),
+  });
+  await migrateCollectionRecord(newerFakeCollection, t.context.knex);
+
+  const createdRecord = await t.context.knex.queryBuilder()
```

Comment:
Same: can we use collectionPgModel.get() here?

Reply:
https://github.com/nasa/cumulus/pull/2162/commits/7027bf1243c36dac3abc1052c1e7b28bdd60a305